generate_scc.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Format taken from OpenLiberty SCC generation in docker images
+
+unset OPENJ9_JAVA_OPTIONS
+
+SCC_SIZE="50m" # Default size for SCC
+SCC_GEN_RUNS_COUNT=3 # Runs for generating SCC
+
+# Creating a temporary directory for tomcat download
+mkdir -p "$HOME"/tomcat/
+
+# Downloading tomcat
+wget --directory-prefix=/tmp https://downloads.apache.org/tomcat/tomcat-9/v9.0.35/bin/apache-tomcat-9.0.35.tar.gz
+
+# Extracting tomcat 
+tar -xvzf /tmp/apache-tomcat-9.0.35.tar.gz -C "$HOME"/tomcat/
+
+# Removing the tar file
+rm -rf /tmp/apache-tomcat-9.0.35.tar.gz
+
+# Creating base layer first instead of running tomcat server to generate SCC directly
+java -Xshareclasses:name=default_scc,cacheDir="$HOME"/scc/,createLayer -Xscmx$SCC_SIZE -version
+
+# Pointing cache for tomcat runs
+export OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=default_scc,cacheDir=$HOME/scc"
```

Comment:
I'd suggest that the scc path be `/opt/java/.scc` dir. Also would be good to add the ` -Xshareclasses:nonfatal ` option to ensure that the application starts up even there are any issues with the SCC.